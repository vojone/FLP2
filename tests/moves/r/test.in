123
456
789
444 123 222 333
111 456 333 444
111 789 333 444
663
662
661
