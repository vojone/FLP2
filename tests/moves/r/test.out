124
451
781
443 741 922 333
112 852 633 444
111 963 333 444
663
663
662
